about/list_of_features.rst

```diff
@@ -429,12 +435,16 @@ Scripting
 :ref:`C#: <toc-learn-scripting-C#>`
 
 - Packaged in a separate binary to keep file sizes and dependencies down.
-- Uses .NET 6.
+- Supports .NET 6 and 7.
```

Comment:
Should we just say .NET 6 and greater? Otherwise we'll have to modify it again when .NET 8.0 is released on November.

Reply:
We can, yes. As long as it's clear and easy to find, I think.